mobile-widgets/qml/Settings.qml

```diff
@@ -647,12 +654,14 @@ TemplatePage {
 					}
 				}
 				TemplateLabel {
-					enabled: PrefTechnicalDetails.calcceiling
+					visible: PrefTechnicalDetails.calcceiling
```

Comment:
Good move, from what I've seen in #4094, there is no obvious visual difference between enabled and disabled elements (at least in our styling), so it's probably more user friendly to go with hiding inactive elements.

Reply:
The grid model isn't great.

There are a few decent introductions to QML, but all are way too simple and never cover the stuff you really want to see 😞

I have done a lot of trial and error